### DISCUSSION TITLE
why 1.24.1 ipa is twice as large compared to 1.24.0?

### DISCUSSION THREAD

**Discussion by vdualb** [aeharding/voyager#954] [Q&A] [answered]
why 1.24.1 ipa is twice as large compared to 1.24.0? apk has the same size

**Comment by aeharding** [comment]
It is due to the revenuecat library for in app purchases.

Unfortunately no lightweight capacitor plugin exists. If one did Voyager could switch to it :)

**Reply by vdualb** [reply]
But android build didn’t increase in size at all, it was build without revenuecat?

**Reply by aeharding** [reply]
There's different native dependencies. You should ask revenue cat

**Reply by aeharding** [reply, marked as answer]
Oh and yes- the build uploaded to the Github Release for Android is without revenue cat (it's the F-droid build)

**Comment by aeharding** [comment]
This should be resolved in 1.26.0.
